core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -570,18 +572,30 @@ private boolean applyRename(RenameEntry entry) {
     mInodeStore.addChild(newParent, inode);
     inode.setParentId(newParent);
     mInodeStore.writeInode(inode);
-    updateLastModified(oldParent, entry.getOpTimeMs());
-    updateLastModified(newParent, entry.getOpTimeMs());
+    updateLastModifiedAndChildCount(oldParent, entry.getOpTimeMs(), -1);
+    updateLastModifiedAndChildCount(newParent, entry.getOpTimeMs(), 1);
     return true;
   }
 
-  private void updateLastModified(long id, long opTimeMs) {
-    try (LockResource lr = mInodeLockManager.lockLastModified(id)) {
-      MutableInode<?> inode = mInodeStore.getMutable(id).get();
-      if (inode.getLastModificationTimeMs() < opTimeMs) {
-        inode.setLastModificationTimeMs(opTimeMs);
-        mInodeStore.writeInode(inode);
+  /**
+   * Updates the last modified time (LMT) for the indicated inode directory, and updates its child
+   * count.
+   *
+   * If the inode's LMT is already greater than the specified time, the inode's LMT will not be
+   * changed.
+   *
+   * @param id the inode to update
+   * @param lastModifiedMs the new last modified time
```

Comment:
why did we rename the param?

Reply:
no big reason, just to match the method name better

Reply:
I liked the previous one because it matched the field name and was less ambiguous (since there are two `lastModifiedMs`

Reply:
switched back